server/prisma/migrations/20240711013942_/migration.sql

```diff
@@ -0,0 +1,36 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Token` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "Token" DROP CONSTRAINT "Token_userId_fkey";
+
+-- DropTable
+DROP TABLE "Token";
+
```

Comment:
Why do we have this?  If you were dropping an existing table in your DB, the code is one-time used.  We can delete them after the table is dropped.

Reply:
Yeah, Prisma takes care of building up the database base on the schema I write, these are migration files based on the changes I do on my schema, the actual tables are built on Postgres